fireshape/control.py

```diff
@@ -261,12 +283,14 @@ def __init__(self, mesh, bbox, orders, levels, fixed_dims=[],
                 univariate B-splines
         fixed_dims: dimensions in which the deformation should be zero
 
-        boundary_regularities: how fast the splines go to zero on the boundary for each dimension
-                               [0,..,0] means that they don't go to zero
-                               [1,..,1] means that they go to zero with C^0 regularity
-                               [2,..,2] means that they go to zero with C^1 regularity
+        boundary_regularities: how fast the splines go to zero on the boundary
+                               for each dimension
+                               [0,..,0] : they don't go to zero
+                               [1,..,1] : they go to zero with C^0 regularity
+                               [2,..,2] : they go to zero with C^1 regularity
         """
-        self.boundary_regularities = [o-1 for o in orders] if boundary_regularities is None else boundary_regularities 
+        self.boundary_regularities = [o - 1 for o in orders] \
+            if boundary_regularities is None else boundary_regularities
```

Comment:
We should `assert` that `boundary_regularities` is compatible with `orders`. E.g.: linear splines (`order = 2`) cannot have C^2 regularity.

Reply:
I agree, but I don't think that is related to this Pull request, we are just doing formatting here.